Vagrantfile

```diff
@@ -34,7 +34,10 @@ $puppetrun=<<SHELL
     rm -rf /etc/puppet/$(echo ${m#/vagrant/})
   done
 
-  puppet config set certname vagrant-base-dev.vagrant.local
+  if [ -z "$HIMLAR_CERT_NAME" ]; then
+    HIMLAR_CERT_NAME=vagrant-base-dev.vagrant.local
+  fi
+  puppet config set certname "$HIMLAR_CERT_NAME"
```

Comment:
We'll need to provide a default here ... so we don't require setting the env. Also, I'd drop the second underscore, so that we get HIMLAR_CERTNAME which will be more inline with the puppet setting